README.md

```diff
@@ -0,0 +1,48 @@
+# Extole Android SDK
+
+## Setup
+
+1. Clone this repository
+2. Open it with Android Studio
+
+## Using Extole SDK
+
+### Initializing
+
+```
+val extole = Extole.Builder("https://extole-monitor-android.extole.io/")
+            .withAppName("extole-mobile-test")
+            .addAppData("version", "1.0")
+            .withSandbox("prod-test")
+            .withDebugEnabled(true)
+            .build()
+```
+
+### Sending events
+
+```
+extole.event("purchase", mapOf(Pair("partner_user_id", "123"), Pair("cart_value", "120.3")))
```

Comment:
120.30

Future: This looks clumsy (I wonder if we should have an event builder)

Reply:
changed.
Builder - probably yes, let's discuss